### PR TITLE
say how to remove password from guest user

### PR DESCRIPTION
**Pull request by mtbc**
Staged at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/sysadmins/server-upgrade.html. Documents https://github.com/openmicroscopy/openmicroscopy/pull/5526.

### CODE REVIEW

File: omero/sysadmins/server-upgrade.rst [hunk 1]
```diff
@@ -272,6 +272,26 @@ run only on the specific version before the next upgrade script.
...
+.. parsed-literal::
+
+    $ psql -h localhost -U **db_user** **omero_database** < sql/psql/|current_dbver|/allow-guest-user-without-password.sql
+
+This can be done at any time during the OMERO 5.4 series.
+
+.. note::
+
+    The above script assumes that the `guest` user has an ID of 1. In
+    the unusual case of that not being so then the script will do
```
Comment (hflynn) [inline omero/sysadmins/server-upgrade.rst]: This is a bit of a mouthful, could we just say "If that is not true then the script..."

Comment (mtbc) [reply omero/sysadmins/server-upgrade.rst]: I *think* it's very unlikely so I don't want this note to worry readers. I'll figure a way to simplify.

**Comment by hflynn** [comment]
👍

**Comment by dominikl** [comment]
Looks good 👍

**Comment by hflynn** [comment]
In light of https://github.com/openmicroscopy/www.openmicroscopy.org/pull/144 maybe this should be amended to cover both aspects of the workaround. I'll hold off merging for now.

**Pull request merged**